pandas/core/internals/managers.py

```diff
@@ -1841,6 +1841,37 @@ def _consolidate_inplace(self) -> None:
             self._known_consolidated = True
             self._rebuild_blknos_and_blklocs()
 
+    # ----------------------------------------------------------------
+    # Concatenation
+
+    @classmethod
+    def concat_horizontal(cls, mgrs: list[Self], axes: list[Index]) -> Self:
+        """
+        Concatenate uniformly-indexed BlockManagers horizontally.
+        """
+        offset = 0
+        blocks: list[Block] = []
+        for i, mgr in enumerate(mgrs):
```

Comment:
Looks like `i` isn't being used here?

```suggestion
        for mgr in mgrs:
```